vector/src/main/java/im/vector/app/features/crypto/verification/choose/VerificationChooseMethodController.kt

```diff
@@ -45,9 +45,22 @@ class VerificationChooseMethodController @Inject constructor(
         val host = this
 
         if (state.otherCanScanQrCode || state.otherCanShowQrCode) {
+            var scanCodeInstructions: String
+            var scanOtherCodeTitle: String
+            var compareEmojiSubtitle: String
+            if (state.isMe) {
+                scanCodeInstructions = host.stringProvider.getString(R.string.verification_scan_self_notice)
```

Comment:
Sorry, I did not noticed it before, but there is no need to use `host.` here. It only useful if you want to use the host member from the Epoxy item DSL, for instance inside `bottomSheetVerificationNoticeItem {}` block.
It's not a big deal at all, it's OK to merge the PR like that, I can do the clean up later.

Reply:
(Fixed in 07d6eaacf95acfec165dce94e567ae89db8e3f14)

Reply:
ah - I see, I did wonder what that was all about, but figured it was best to be consistent anyhow